apps/redi-talent-pool/src/components/organisms/jobseeker-profile-editables/EditableEducation.tsx

```diff
@@ -190,9 +191,29 @@ export function JobseekerFormSectionEducation({
     })
   }
 
+  const validationSchema = Yup.object().shape({
```

Comment:
If we're gonna start doing a validation schema:
1) Can you validate these rules with @miriam-aha ? This is definitely new functionality and should be reviewed with the PO
2) Think carefully about the UX of the validation messages. `EditableJobPostings.tsx` can be worth checking out for an example of less exclamation-mark heavy validation messages.
3) I don't see the point of adding fields that don't require any validation to the validation schema (meaning all that don't have a `.required()` tag. What do we think about removing them?

Reply:
Hey, please let me know what you need from my side.